src/Admin/Customizer/CustomControls/WP_Customize_Integration_Repeater_Control.php

```diff
@@ -609,7 +655,7 @@ public function render_content()
         Apparently, it is automatically inserted by customizer*/
         ?>
         <div class="mo-integration__add_new">
-            <button type="button" class="button mo-add-new-integration">
+            <button type="button" class="button mo-integration-widget-actions">
```

Comment:
Why was the class name changed?

Reply:
@ShoboySnr did yoy miss this?

Reply:
I added a different custom css styling to this.